week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -26,9 +26,9 @@ function mood() {
 
 function greaterThan10() {
   const num = 10;
-  const isBigEnough;
+  let isBigEnough =10;
 
-  if (isBigEnough) {
+  if (isBigEnough ==10) {
     return "num is greater than or equal to 10";
   } else {
     return "num is not big enough";
```

Comment:
Although this code works, the aim of this exercise would be different.
Please revise the instructions again:
```
only change isBigEnough.
Variable isBigEnough needs to evaluate to a boolean
```
How would you modify line `let isBigEnough =10;` in order to make the value type of isBigEnough to Boolean instead of Number?

Hint: You should try to define a logical expression on the left side of `const isBigEnough` that would result in either `true` or `false` value.